TestSwitchApi/TestSwitchApi/Models/DataModels/CandidateDataModel.cs

```diff
@@ -12,5 +14,8 @@ public class CandidateDataModel
         public string LastName { get; set; }
         public string Email { get; set; }
         public string Guid { get; set; }
+
+        [JsonIgnore]
```

Comment:
Was this just until you managed to solve the error we talked about on slack?

I think what we are really seeing is that we are using the same model for the API response and the DB model.
And we are starting to find that we are wanting slightly different things in those cases.

As we said - ideal fix is perhaps to split into different models, but for now changing the JSON mapping config seems like a good option! :)